build-tools/xaprepare/xaprepare/Steps/Step_PrepareExternal.cs

```diff
@@ -14,13 +14,18 @@ protected override async Task<bool> Execute (Context context)
 		{
 			var nuget = new NuGetRunner (context);
 
-			if (!await NuGetRestore (nuget, context.XASolutionFile)) {
-				return false;
+			foreach (var solutionFile in context.XASolutionFile) {
+				if (!await NuGetRestore (nuget, solutionFile)) {
+					return false;
+				}
 			}
 
 			Log.StatusLine ();
-			if (!await NuGetRestore (nuget, context.XATestsSolutionFile)) {
-				return false;
+
+			foreach (var solutionFile in context.XASolutionFile) {
```

Comment:
This might be a minor typo:
```suggestion
			foreach (var solutionFile in context.XATestsSolutionFilePath) {
```

Reply:
Fixed!